packages/salesforcedx-vscode-replay-debugger/src/core/logContextUtil.ts

```diff
@@ -6,13 +6,11 @@
  */
 
 import * as fs from 'fs';
-import { EOL } from 'os';
 
 export class LogContextUtil {
   public readLogFile(logFilePath: string): string[] {
     try {
-      const fileContent = fs.readFileSync(logFilePath).toString('utf-8');
-      return fileContent.split(EOL);
+      return fs.readFileSync(logFilePath, 'utf-8').split('\n');
```

Comment:
When I downloaded a debug log to my Windows, the line breaks were actually `\n` instead of the expected `\r\n`. The latter is what is provided by `os.EOL`, but it wasn't working due to this anomaly. 

![unexpectedeolonwindows](https://user-images.githubusercontent.com/7286437/36624802-ee386986-18c8-11e8-9942-0a4b32146b4b.gif)

I think hardcoding `\n` here is safe for other platforms. The alternative is https://www.npmjs.com/package/eol and it would cost more in memory:

`import * as eol from 'eol';`
`eol.split(eol.auto(fs.readFileSync(logFilePath, 'utf-8')));`

Reply:
I think that's expected since our servers are the ones that generate the log file and the servers are running on *nix. The OS of the user is not taken into account when we generate the logs.

However, how about using a safer regex to split like `"a\nb\r\nc".split(/\r?\n/)` from https://stackoverflow.com/a/21895354 

This will take care of both *nix and Windows. It doesn't take care of classic Mac which used only `\r` but I don't think that is relevant anymore.

Also, this handles the case in case someone were to edit the file by hand (on a Windows machine) which would actually put \r\n in one some lines.